server/auth/types/basic/routes.ts

```diff
@@ -194,8 +197,12 @@ export class BasicAuthRoutes {
             sessionStorage.tenant = selectTenant;
           }
           this.sessionStorageFactory.asScoped(request).set(sessionStorage);
-
-          return response.ok({
+          return response.redirected({
+            headers: {
+              location: `${this.coreSetup.http.basePath.serverBasePath}${
+                request.url.path?.split('?')[1]
```

Comment:
Is it possible that `request.url.path?` is null? What will happen to this `request.url.path?.split('?')[1]` then, could it be `'null'` or `'undefined'` rather than `''`?

Reply:
Looks like this change caused a integ test to error, please update the test case as well

Reply:
sure will do that